guides/common/modules/proc_synchronizing-ubuntu-expanded-security-maintenance-content.adoc

```diff
@@ -0,0 +1,78 @@
+[id="Synchronizing-Ubuntu-Expanded-Security-Maintenance-content_{context}"]
+= Synchronizing Ubuntu Expanded Security Maintenance content
+
+Canonical provides Ubuntu https://ubuntu.com/security/esm[Expanded Security Maintenance] (ESM) repositories to their paying customers.
+Use {Project} to synchronize ESM repositories from Canonical to distribute content to your hosts running Ubuntu 14.04 and Ubuntu 16.04.
+
+Extract the ESM repository information from an existing host running Ubuntu 14.04 or 16.04 once by hand to enable synchronization to {Project}.
```

Comment:
```suggestion
```
It isn't the extraction that enables the sync, but the "subscription attachment", probably.

Reply:
Removed.